.cirrus.yml

```diff
@@ -22,11 +22,18 @@ task:
       test_script: ./script/incremental_build.sh test
     - name: analyze
       script: ./script/incremental_build.sh analyze
-    - name: build-apks+java-test
+    - name: build-apks+java-test+drive-examples
       env:
         matrix:
           PLUGIN_SHARDING: "--shardIndex 0 --shardCount 2"
           PLUGIN_SHARDING: "--shardIndex 1 --shardCount 2"
+      create_device_script:
+        echo no | avdmanager create avd --force \
```

Comment:
Is this slow? is so maybe worth keeping it as part of the docker image?

Reply:
According to https://medium.com/cirruslabs/speeding-up-android-builds-with-cirrus-ci-9e20ab7c5150 it's pretty fast but also I don't see a downside to putting it in the Dockerfile so I'll try that

Reply:
If it's fast probably simpler to keep it here